index.bs

```diff
@@ -872,27 +872,30 @@ Advisory Board (AB)</h4>
 <h5 id=ab-role>
 Role of the Advisory Board</h5>
 
-	The <dfn export lt="Advisory Board|AB">Advisory Board</dfn> provides ongoing guidance to the Team
+	The <dfn export lt="Advisory Board|AB">Advisory Board</dfn>
+	is elected by the Membership to advance the effectiveness of W3C as a community by:
```

Comment:
```suggestion
	is elected by the Membership to advance the effectiveness of W3C as a standards development organization (<abbr title="standards development organization">SDO</abbr>) by:
```

Reply:
Rationale: why is the membership here? To work on specs. What does W3C have to be effective at? At being a venue for standards work.